components/Footer/Footer.vue

```diff
@@ -27,48 +27,53 @@
             </a>
           </div>
           <div class="copyright__container">
-            Copyright 2020 - Mercedes-Benz.io & BCG
+            {{ localI18n['footer.copyright'] }}
           </div>
         </div>
         <div class="item__section__container">
           <div class="item__section">
             <h6 class="item__header">
-              <nuxt-link class="item__header__link" to="/">Home</nuxt-link>
+              <nuxt-link class="item__header__link" to="/">
+                {{ localI18n['footer.home'] }}
+              </nuxt-link>
             </h6>
           </div>
           <div class="item__section">
             <h6 class="item__header">
               <nuxt-link
                 class="item__header__link"
                 to="/doar-telemoveis-tablets"
-                >Iniciativa #darvozAQuemNaoTem</nuxt-link
               >
+                {{ localI18n['footer.donateInitiative'] }}
+              </nuxt-link>
             </h6>
             <p>
-              <nuxt-link class="item" to="/doar-telemoveis-tablets#como-doar"
-                >Como doar</nuxt-link
-              >
+              <nuxt-link class="item" to="/doar-telemoveis-tablets#como-doar">
+                {{ localI18n['footer.donateInitiative.link1'] }}
+              </nuxt-link>
             </p>
             <p>
               <nuxt-link
                 class="item"
                 to="/doar-telemoveis-tablets#pontos-recolha"
-                >Pontos de recolha</nuxt-link
               >
+                {{ localI18n['footer.donateInitiative.link2'] }}
+              </nuxt-link>
             </p>
           </div>
           <div class="item__section">
             <h6 class="item__header">
               <nuxt-link
                 class="item__header__link"
-                to="/carris-mensagens-gratis"
-                >Iniciativa #darvozCarris</nuxt-link
+                to="/quadras-santos-populares"
               >
+                {{ localI18n['footer.quadraInitiative'] }}
+              </nuxt-link>
             </h6>
             <p>
-              <nuxt-link class="item" to="/carris-mensagens-gratis#mensagem"
-                >Deixe uma mensagem</nuxt-link
-              >
+              <nuxt-link class="item" to="/quadras-santos-populares">
+                {{ localI18n['footer.quadraInitiative.link'] }}
```

Comment:
we are using different names for the iniatives on the i18n. We should normalize them. Can be done afterwards right?

Reply:
yes when we have the final names